src/scenes/ProgressReports/StatusHistory/StatusHistory.tsx

```diff
@@ -0,0 +1,117 @@
+import { useQuery } from '@apollo/client';
+import {
+  Box,
+  Breadcrumbs,
+  FormControlLabel,
+  Skeleton,
+  Switch,
+  Typography,
+} from '@mui/material';
+import { useToggle } from 'ahooks';
+import { Helmet } from 'react-helmet-async';
+import { flexColumnStyle } from '~/common';
+import { Breadcrumb } from '~/components/Breadcrumb';
+import { idForUrl, useChangesetAwareIdFromUrl } from '~/components/Changeset';
+import { EngagementBreadcrumb } from '~/components/EngagementBreadcrumb';
+import { ReportLabel } from '~/components/PeriodicReports/ReportLabel';
+import { ProjectBreadcrumb } from '~/components/ProjectBreadcrumb';
+import { Navigate } from '~/components/Routing';
+import { Error } from '../../../components/Error';
+import {
+  ProgressReportDetailDocument,
+  ProgressReportDetailFragment,
+} from '../Detail/ProgressReportDetail.graphql';
+import { WorkFlowEventHistory } from '../Detail/WorkflowEventsHistory';
+
+export const StatusHistory = () => {
+  const { id, changesetId } = useChangesetAwareIdFromUrl('reportId');
+  const [checked, set] = useToggle();
+
+  const { data, error } = useQuery(ProgressReportDetailDocument, {
+    variables: {
+      id,
+      changesetId,
+    },
+  });
+  if (error) {
+    return (
+      <Error page error={error}>
+        {{
+          NotFound: 'Could not find status history',
+          Default: 'Error loading status history',
+        }}
+      </Error>
+    );
+  }
+
+  const report = data?.periodicReport as
+    | ProgressReportDetailFragment
+    | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- safety check. It's possible with manual user input.
+  if (report && report.__typename !== 'ProgressReport') {
+    return <Navigate to="/" replace />;
+  }
+
+  const engagement = report?.parent;
+
+  return (
+    <div css={{ overflowY: 'auto' }}>
+      <Box component="main" sx={{ ...flexColumnStyle, p: 4 }}>
```

Comment:
Ah here's why that change was made. Instead:
```suggestion
      <Box component="main" css={flexColumn} sx={{ p: 4 }}>
```